terraform/main.tf

```diff
@@ -54,6 +63,13 @@ provider "google-beta" {
   project = var.gcp_project
 }
 
+provider "aws" {
+  # aws_s3_bucket resources will be created in the region specified here
+  # https://github.com/hashicorp/terraform/issues/12512
```

Comment:
This issue doesn't seem relevant? It's only talking specifically about `us-east-1`, not in general.

Reply:
Maybe the usage of "here" is ambiguous. I meant that the `region` field of the `aws` provider is used to determine the region of the bucket, because I found it surprising that Terraform expressed it this way since the [underlying API[(https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateBucket.html#API_CreateBucket_RequestSyntax) allows explicitly specifying the bucket's region. I'll make the comment more clear.